src/main/java/seedu/address/commons/core/LogsCenter.java

```diff
@@ -56,7 +66,15 @@ public static Logger getLogger(String name) {
      */
     public static <T> Logger getLogger(Class<T> clazz) {
         requireNonNull(clazz);
-        return getLogger(clazz.getSimpleName());
+        // The BASE_LOGGER_NAME acts as a common prefix for all application loggers,
+        // organizing them based on functionality or module.
+        // The dot (".") serves as a separator, reflecting the package structure and providing a
+        // hierarchical representation.
```

Comment:
Not applicable here, as we are using just the class name (i.e., we are not using package names as logger names)